### PR TITLE
Custom separator for create topics

### LINKED ISSUE #208: Allow multiline definition of KAFKA_CREATE_TOPICS
Hi! It would be nice to be able to specify the topics to be created on startup with newlines.

```yaml
    ...
      KAFKA_CREATE_TOPICS: |
        topic1:100:3
        topic2:100:3
        topic3:100:3
        ...
```
docker-compose.yml

### PR DESCRIPTION
**Pull request by sscaling**
Fixes #208

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ Changelog
 
 Kafka features are not tied to a specific kafka-docker version (ideally all changes will be merged into all branches). Therefore, this changelog will track changes to the image by date.
 
+TBC
+---
+
+-	Issue #208 - Add `KAFKA_CREATE_TOPICS_SEPARATOR` to allow custom input, such as multiline YAML.
```
Comment (jaceklaskowski) [inline CHANGELOG.md]: nit: spaces between `-` and "Issue" + s/multiline/multi-line

File: README.md [hunk 2]
```diff
@@ -72,6 +72,8 @@ Here is an example snippet from ```docker-compose.yml```:
 
 ```Topic 1``` will have 1 partition and 3 replicas, ```Topic 2``` will have 1 partition, 1 replica and a `cleanup.policy` set to `compact`.
 
+If you wish to use multi-line YAML or some other delimiter between your topic definitions, then can override the default `,` separator by specifying the `KAFKA_CREATE_TOPICS_SEPARATOR` variable. i.e. `"$$'\n"'` would be a newline following docker-compose escaping rules, and [ANSI-C](https://www.gnu.org/software/bash/manual/html_node/ANSI_002dC-Quoting.html) quoting.
```
Comment (jaceklaskowski) [inline README.md]: 1. Remove "then can" before "override"
2. `i.e.` should rather be "For example" on a new line to make the example stand out.
3. s/variable/environment variable

File: test/test.create-topics-custom-separator.kafka.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+			echo "$TOPIC topic not created"
+			return 1
+		fi
+	done
+
+	return 0
+}
+
+# mock the netstat call as made by the create-topics.sh script
+function netstat() { echo "1 2 3 :$KAFKA_PORT"; }
```
Comment (jaceklaskowski) [inline test/test.create-topics-custom-separator.kafka.sh]: That's very clever! I have never thought about such mocking.

File: CHANGELOG.md (hunk 1 above)
Comment (sscaling) [reply CHANGELOG.md]: > nit: spaces between - and "Issue"

It's a tab - and it's [markdownfmt](https://github.com/shurcooL/markdownfmt) auto-formatted and consistent with the rest of the file.

I'll fix the "multi-line"

File: test/test.create-topics-custom-separator.kafka.sh (hunk 3 above)
Comment (sscaling) [reply test/test.create-topics-custom-separator.kafka.sh]: needs must.

**Pull request merged**